tests/unittest/_torch/executor/test_py_scheduler.py

```
@@ -2073,6 +2073,31 @@ def test_lora_doesnt_fit(self):
+        kv = MockKVCacheManager(num_free_blocks=100, blocks_per_request=5)
+        peft = MockPeftCacheManager(max_pages=25, pages_per_request=10)
+        scheduler = PyCapacityScheduler(
+            max_num_requests=4,
+            kv_cache_manager=kv,
+            peft_cache_manager=peft,
+            scheduler_policy=CapacitySchedulerPolicy.MAX_UTILIZATION,
+        )
+        r0 = _make_request(0, lora_task_id=1)
+        r1 = _make_request(1, lora_task_id=2)
+        r2 = _make_request(2, lora_task_id=3)
+        fitting, disagg, paused = scheduler.schedule_request([r0, r1, r2])
```

Ruff (0.15.12)

[warning] Unpacked variable `disagg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)